lib/events/stream.go

```diff
@@ -386,16 +386,21 @@ func (s *ProtoStream) Done() <-chan struct{} {
 
 // EmitAuditEvent emits a single audit event to the stream
 func (s *ProtoStream) EmitAuditEvent(ctx context.Context, event apievents.AuditEvent) error {
+	messageSize := event.Size()
+	if messageSize > MaxProtoMessageSizeBytes {
```

Comment:
not related to PR but I was always curios why the max limit for Proto message was set to  64KB ?